docs/CSS.md

```diff
@@ -0,0 +1,98 @@
+### 1. `flex`
+* *elements:* none
+* *inherited:* no 
+
+#### Values
+
+* `<flex-grow>` -- defines the flex-grow of the flex item
+* `<flex-shrink>` -- defines the flex-shrink of the flex item
+* `<flex-basis>` -- defines the flex-basis of the flex item
+* `none` -- this keyword computes to 0 0 auto;
+
+### 2. `margin`
+* *elements:* all elements except elements with table display types other than table-caption, table and inline-table.
+* *inherited:* no
+
+#### Values
+
+* `<length>` -- specifies a fixed width
+* `<percentage>` -- relative to the width containing the block
+*  `auto` -- auto is replaced by some suitable value 
+
```

Comment:
Technically, `margin` is another of those _composites_, but it's a little easier to tackle. What are the properties that roll up into `margin`...? It would also be helpful to document the different types of length units, at least as a list. And pay special attention to what the percentage value is a percentage _of_...